tests/test_validators.py

```
@@ -618,3 +626,175 @@ def test_boundary_values_polling(self):
+
+    def test_valid_api_key(self):
+        """Valid 32-char API key should pass."""
+        key = "AgMJSwGyNpi1YXXfCcQ4qpxcPo-hif0o"
+        is_valid, sanitized, error = validate_api_key(key)
+        assert is_valid is True
+        assert sanitized == key
+        assert error is None
+
+    def test_key_too_short(self):
+        """Short API key should fail."""
+        is_valid, sanitized, error = validate_api_key("abc123")
```

Ruff (0.15.9)

[warning] Unpacked variable `sanitized` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)